common/app/views/support/GetClasses.scala

```diff
@@ -192,6 +194,7 @@ object GetClasses {
     BreakingPalette -> "fc-container--breaking-palette",
     EventPalette -> "fc-container--event-palette",
     EventAltPalette -> "fc-container--event-alt-palette",
+    SpecialReportAltPalette -> "fc-container--special-report-alt-palette",
```

Comment:
There is a WIP PR to add styles for this class: https://github.com/guardian/interactive-atom-container-colours/pull/4